tests/Bugsnag.Tests/Payload/DeviceTests.cs

```diff
@@ -21,12 +22,21 @@ public void SpecifyHostName()
     }
 
     [Fact]
-    public void SpecifyTime()
+    public void IncludesTime()
     {
       var device = new Device("hostname");
       Assert.NotNull(device["time"]);
     }
 
+    [Fact]
+    public void IncludesRuntimeVersions()
+    {
+      var device = new Device("hostname");
+      var runtimeVersions = (Dictionary<string, string>) device["runtimeVersions"];
+      Assert.NotNull(runtimeVersions["dotnet"]);
```

Comment:
Can you add a regex test to ensure it contains `\d+\.` 4 times? This is what was tested in other notifiers.